models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php

```diff
@@ -774,8 +774,9 @@ public function getColumnKeys()
 
     /**
      * @param DataObject\ClassDefinition $class
+     * @param array $params
      */
-    public function classSaved($class)
+    public function classSaved($class, $params = [])
```

Comment:
Technically, this is a BC break, too, as this class/method is neither `@internal` nor `final`, so someone *could* extend it, which would break after this change.